packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -253,6 +253,9 @@
     "sign_up_page_producer_hint": "Producer/brand",
     "sign_up_page_producer_error_empty": "Please enter a producer or a brand name",
     "sign_up_page_subscribe_checkbox": "I'd like to subscribe to the Open Food Facts newsletter (You can unsubscribe from it at any time)",
+    "sign_up_page_user_name_already_used": "The user name already exists, please choose another username.",
+    "sign_up_page_email_already_exists": "already exists, login to the account or try with another email.",
+    "sign_up_page_provide_valid_email": "Please provide a valid email adress.",
```

Comment:
```suggestion
    "sign_up_page_provide_valid_email": "Please provide a valid email address.",
```

Reply:
oh okay, I will change that! xD did not notice that !.